saleor/dashboard/customer/views.py

```diff
@@ -41,6 +43,28 @@ def customer_details(request, pk):
     return TemplateResponse(request, 'dashboard/customer/detail.html', ctx)
 
 
+@staff_member_required
+@permission_required('userprofile.edit_user')
+def customer_edit(request, pk=None):
+    if pk:
```

Comment:
Don't we have separate views for editing and creating in other modules?

Reply:
Right now in saleor, we have this mixed. For example, shipping_methods (and other former formsets) has one view for edit and create. This is sth I wanted to discuss with you.

Reply:
So I think that it's much better to have separate views and I would consider those that do both creating and editing a technical debt.

Reply:
I agree. Separate views for edit and create actions are more readable. Even if they are very similar.